### DISCUSSION TITLE
https://plugin-sass-palette.vuejs.press/zh/guide.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#546] [Announcements]
# https://plugin-sass-palette.vuejs.press/zh/guide.html

这个插件基本上是面向插件和主题开发的。 ID 首先，你应该了解此插件的设计目标是提供跨越插件和主题的支持。(而并不像官方插件仅面向主题)。 我们提供了 id 选项来执行此操作，并且使用具有相同 ID 的此插件 (通过调用 useSassPalette) 不会有任何副作用。 此外，所有别名和模块名称都有一个 ID 前缀。 这将允许你: 使用相同的 ID ...

https://plugin-sass-palette.vuejs.press/zh/guide.html

**Comment by lingdu990130556** [comment]
〇°
